### ISSUE TITLE
add time formats tooltip to the !remind command

### ISSUE THREAD

**Issue by swordysrepo** [python-discord/bot#1655]
https://imgur.com/a/tYsTD7S
this suggestion is for adding documentation to the tooltip 
suggesting to have a tooltip on which time format is accepted when setting a reminder . 

started with doing` !remind new 12 api ` and it didn't accept the format. (i was guessing what this would do, because otherwise i would check the code on the github to find the formats) , and then i assumed that a `h` format could be used and it worked . 

note:
to quickly get more information in tooltips while on the discord i used `!remind new `and `!remind add` and they also don't provide which formats are accepted

**Comment by anand2312** [comment]
This is done in the help command
![preview](https://why.life-is-pa.in/531dl4Wji.png)
which links to [strftime.org](https://strftime.org) but I do think updating the help command to instead show a few examples would be nice.

**Comment by wookie184** [comment]
Yeah, I think including the information in the help command is enough here, although I don't think the current link to strftime.org is actually very helpful in this case, as it actually uses a custom duration parser, https://github.com/python-discord/bot/blob/main/bot/converters.py#L296. There is overlap as it's inspired from strftime, but I don't think looking at that page makes how it works clear.
It could probably be edited to show something like this (mostly just copied from the converter docstring):
> The expiration duration supports the following symbols for each unit of time:
    - years: `Y`, `y`, `year`, `years`
    - months: `m`, `month`, `months`
    - weeks: `w`, `W`, `week`, `weeks`
    - days: `d`, `D`, `day`, `days`
    - hours: `H`, `h`, `hour`, `hours`
    - minutes: `M`, `minute`, `minutes`
    - seconds: `S`, `s`, `second`, `seconds`
>    
>    For example, to set a reminder that expires in 3 days and 1 minute, you could do `!remind new 3d1M Do something`.

It's a bit large, but probably useful information.

**Comment by TizzySaurus** [comment]
Have just discussed this with @ChrisLovering in #dev-core. The decision was to edit the docstring of both the `remind` group and `remind new` command to have the suggested "documentation" of the supported time types.

I'll work on this shortly.
